src/test/scala/uk/gov/hmrc/perftests/example/AccessibilityStatementFrontendRequests.scala

```diff
@@ -30,8 +30,8 @@ object AccessibilityStatementFrontendRequests extends ServicesConfiguration {
       .get(s"$baseUrl/accessibility-statement/$${serviceName}")
       .resources(
         http("font-bold").get(s"$baseUrl/accessibility-statement/hmrc-frontend/assets/govuk/fonts/bold-b542beb274-v2.woff2"),
-        http("css").get(s"$baseUrl/accessibility-statement/hmrc-frontend/assets/hmrc-frontend-5.11.1.min.css"),
-        http("js").get(s"$baseUrl/accessibility-statement/hmrc-frontend/assets/hmrc-frontend-5.11.1.min.js"),
+        http("css").get(s"$baseUrl/accessibility-statement/hmrc-frontend/assets/all-govuk-and-hmrc.scss"),
```

Comment:
it's not the same size, but I think more than size it's just that it's doing another network request which is good enough to simulate getting minified css

Reply:
I couldn't find another file with a similar size, I did try setting both JS and CSS to point to the same file and the tests passed but not sure if its a good idea in terms of a performance test